### PR TITLE
 London Class 7 - Deniz Kelemence- Javascript Core 1 - Week 2

### PR DESCRIPTION
**Pull request by ASLANDENIZ**
# Your Details

- Your Name: Deniz Kelemence
- Your City: London
- Your Slack Name: Deniz Kelemence

# Homework Details

- Module: JavaScript
- Week: 2

### CODE REVIEW

File: exercises/C-comparison-operators/exercise.js [hunk 1]
```diff
@@ -7,14 +7,14 @@
 
 var studentCount = 16;
 var mentorCount = 9;
-var moreStudentsThanMentors; // finish this statement
+var moreStudentsThanMentors = studentCount > mentorCount; // finish this statement
 
 var roomMaxCapacity = 25;
-var enoughSpaceInRoom; // finish this statement
+var enoughSpaceInRoom = roomMaxCapacity === studentCount + mentorCount // finish this statement
```
Comment (NeginDn) [inline exercises/C-comparison-operators/exercise.js]: I suggest you change this line to:
var enoughSpaceInRoom = studentCount + mentorCount <= roomMaxCapacity;
Although your code is working fine with the test, if number of students and mentors would be less than 25, the answer to the question of " Is there enough space in the room for all students and mentors? " would return false with your code

File: exercises/C-comparison-operators/exercise.js [hunk 2]
```diff
@@ -7,14 +7,14 @@
...
 
 var personA = "Daniel";
 var personB = "Irina";
-var sameName; // finish this statement
+var sameName = personA ==! personB;// finish this statement
```
Comment (NeginDn) [inline exercises/C-comparison-operators/exercise.js:17]: Also here, if you check it with the same names(both Daniel for example) the return value still would be false. I suggest you change it to: `var sameName = personA === personB;`

File: exercises/F-logical-operators/exercise2.js [hunk 3]
```diff
@@ -5,7 +5,23 @@
...
+  return number < 0;
+
+}
+
+function isBetween5and10(number) {
+  return 5 < number < 10;
+}
+
+function isShortName(name) {
+  return name.length < 6;
```
Comment (NeginDn) [inline exercises/F-logical-operators/exercise2.js:18]: I think As Daniel has 6 Characters, in order to get the expected result, you need to have a bigger number here

File: exercises/G-conditionals-2/exercise-3.js [hunk 4]
```diff
@@ -8,7 +8,18 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >= 80) {
```
Comment (NeginDn) [inline exercises/G-conditionals-2/exercise-3.js:12]: Just to let you know if you if block contains just one line, you don't have to use {}, You can write this code shorter this way like :
 ```
 if ((mark) >= 80) return "A";
  else if (mark < 80 && mark > 60) return "B";
  else if (mark < 60 && mark > 50) return "C";
  else return "F";
}

```

**Review by NeginDn — COMMENTED** [review]
Hi Deniz,
Great job with the homework, your codes are very clean, I suggested some minor changes.

**Review by NeginDn — COMMENTED** [review]
Hi Deniz,
Great job with the homework, your codes are very clean, I suggested some minor changes.

**Comment by ASLANDENIZ** [comment]
> Hi Deniz,
> Great job with the homework, your codes are very clean, I suggested some minor changes.

Hello Negin, 

Thank you so much for recommendations, I did make the changes. :)

File: exercises/G-conditionals-2/exercise-3.js [hunk 5]
```diff
@@ -8,7 +8,13 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >= 80) return "A";
+  else if (60 < mark < 80) return "B";
```
Comment (nPelvin) [inline exercises/G-conditionals-2/exercise-3.js:13]: Hey Deniz,

Hope you are well!
This code looks great. You could even simplify it further by changing it to ** else if mark >60 return "B"; **
The reason you wouldn't need to check if it is< 80 is that it has already checked on line 12 before passing down to the else statement. 

You can also make a similar change to line 14.

Hope that makes sense!

Nigel

Comment (ASLANDENIZ) [reply exercises/G-conditionals-2/exercise-3.js]: Hello Nigel,

I'm well, hope you are good too. You are right, I just edited my code, thank you so much.

**Pull request closed without merging**